src/transactions.rs

```diff
@@ -41,8 +48,7 @@ pub struct Transaction<L: Ledger> {
     /// The uids of the outputs of this transaction for which memos have not yet been posted.
     pending_uids: HashSet<u64>,
     /// A receiver memo for each output, except for burned outputs.
```

Comment:
I think this comment should be moved up to `SignedMemos::memos`, since it explains the somewhat surprising type `Vec<Option<ReceiverMemo>>`